addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -236,7 +240,7 @@ private synchronized void refreshData() {
 		try {
 			refreshDeviceData();
 			if (connectionEstablished) {
-				updateStatus(ThingStatus.ONLINE);
+				if (!getThing().getStatus().equals(ThingStatus.ONLINE)) updateStatus(ThingStatus.ONLINE);
```

Comment:
The main reason why I added it was #356
Also as the status update message come as INFO, I found it bit polluting the logs.

Maybe I'll propose a PR to only have the change of status as INFO in the log, and status updates that remain the same status as DEBUG in the logs